mindsdb/integrations/handlers/zotero_handler/zotero_handler.py

```diff
@@ -0,0 +1,227 @@
+import os
+import ast
```

Comment:
@ElinaKapetanaki Please change this import to from `mindsdb_sql.parser import ast` and try again?

Reply:
Alright, will do. Is this the cause of the "not installed zotero handler" problem?

Reply:
I believe the handler has become unusable due to this error. Let me know if this fixes your issue.

Reply:
Great!! The `CREATE DATABASE` statement worked now! I will work on fixing a few things more and will update this pr.

Reply:
The queries work!! Made quite a few changes to my code. I want to make it a bit cleaner and will update the pr with also some pictures of the functionality.

Reply:
@ElinaKapetanaki Sounds great! Just mark the PR as ready for review whenever you are ready and I will take a look.